### PR TITLE
Improve blog teaser design

### PR DESCRIPTION
**Pull request by IshaDakota**
#418 

IMO the best way to handle this. Sets height on the teasers (desktop only) - when title is longer than 2 lines, overflow is hidden (we should change those titles).

fireshot capture 20 - gizra s blog - http___0 0 0 0_9000_blog_

### CODE REVIEW

**Comment by IshaDakota** [comment]
@amitaibu please review.

**Comment by amitaibu** [comment]
c5ad0124-2433-11e6-8200-115cd65fd5a3_png__2540x3570__

**Comment by RachelBaram** [comment]
I assume I was assigned to give a shorter name to the blog? If yes, then how about?
Budget Goggles - a common language & setting expectations OR (If this is not short enough) 
Budget Goggles - setting stakeholder expectations.

**Comment by IshaDakota** [comment]
@RachelBaram see #426

**Comment by IshaDakota** [comment]
let's save this for the discussion on #346, because a smaller base font is going to help this a bit.

**Comment by IshaDakota** [comment]
actually this was pretty quick:
fireshot capture 41 - gizra s blog - http___0 0 0 0_9000_blog_

With this typography, these are the titles that need to be fixed... @RachelBaram want to take a crack? When you're done I'll go through and fix capitalizations (blech):

- Budget Goggles - creating a common language and setting expectations
- How We Achieve Faster Development and Happier Clients by Opening Our GitHub 
- Tipa.li – a Tool for Easily Finding Polio Vaccination Clinics
- Code sample for DrutNet API - connect, load node, save node and upload file

**Comment by IshaDakota** [comment]
@RachelBaram ^^ propose title changes

**Comment by RachelBaram** [comment]
@IshaDakota  My try: 

1. Budget Goggles - a common language and setting expectations (if it needs further editing, then can take our 'a common language and')
2. An Open Github for Faster Development and Happier Clients (if it needs further editing, can write 'dev' instead of 'development')
3. Tipa.li – A tool to find a Polio Vaccination Clinic (if it needs further editing, can take out 'a tool to')
4. Code sample for DrutNet API (I deleted here a lot of this very technical title... does it 'ruin' the meaning?)

**Comment by IshaDakota** [comment]
Sorry for the back and forth on this, but after adjusting to match the larger typography on other pages, it's clear that the right approach here is to open up three lines of text for the title - no editing necessary.

fireshot capture 56 - gizra s blog - http___0 0 0 0_9000_blog_

**Comment by IshaDakota** [comment]
@amitaibu for your review.

**Comment by RachelBaram** [comment]
@IshaDakota The large space below is not pretty. Are we talking about a lot of title changes to decrease it to two lines?

**Comment by IshaDakota** [comment]
@RachelBaram I'm hesitant to shrink the titles to 18px (smaller than the body font), but that would be the easiest resolution. There are quite a few titles that need to be fixed at this size, so two lines isn't really an option, if we leave it at this size

Is there something that would make removing the border more appealing to you and @amitaibu ? That to me seems the best option.

Of course, we could just try a different design altogether.

**Comment by IshaDakota** [comment]
Like this maybe:

fireshot capture 57 - gizra s blog - http___0 0 0 0_9000_blog_

**Comment by IshaDakota** [comment]
Or two across:

fireshot capture 58 - gizra s blog - http___0 0 0 0_9000_blog_

**Comment by IshaDakota** [comment]
Or if you want to get really wild: http://tympanus.net/Development/GridLoadingEffects/index2.html

**Comment by RachelBaram** [comment]
That is really cool! It may not match our somewhat conservative/static
design. I feel comfortable with the two in a row option but let's keep this
in mind when we do the overhaul.

**Comment by IshaDakota** [comment]
Note title change:
fireshot capture 59 - gizra s blog - http___0 0 0 0_9000_blog_

File: app/_scss/_blog.scss [hunk 1]
```diff
@@ -103,12 +100,12 @@
 @include desktop {
 
   .blog {
-    .posts >*:nth-of-type(3n+1) {
-      clear: both;
-    }
+    //.posts >*:nth-of-type(3n+1) {
```
Comment (amitaibu) [inline app/_scss/_blog.scss]: better to remove completely instead of commenting out (can be in a follow up)

**Comment by amitaibu** [comment]
Great one, thanks!

**Pull request merged**